### PR TITLE
add blockstream jade to hardware.md

### PR DESCRIPTION
**Pull request by marc3linho**
Blockstream Jade is also a recommendable hardware wallet / signing device

### CODE REVIEW

File: wallet/hardware.md [hunk 1]
```diff
@@ -25,10 +25,11 @@ This type of set-up does come with additional hurdles that may be more difficult
 
 1.  [BitBox02](#bitbox02)
 2.  [Coldcard](#coldcard)
-3.  [Passport](#passport)
-4.  [Seed Signer](#seed-signer)
-5.  [Specter DIY](#specter-diy)
-6.  [Trezor](#trezor)
+3.  [Jade](#Jade)
```
Comment (BitcoinQnA) [inline wallet/hardware.md:28]: needs to be lower case or the link won't work

File: wallet/hardware.md [hunk 2]
```diff
@@ -82,6 +83,31 @@ This type of set-up does come with additional hurdles that may be more difficult
 
 ***
 
+## Jade
+
+The Blockstream [Jade](https://blockstream.com/jade/) is one of the smallest hardware wallets on the market but offers a lot of features like the support of SeedQR and a built-in battery. Beginners or people who are not interested in the us of stateless devices have the option to use USB, Bluetooth or SeedQR with a stored seed in combination with a virtual secure element (pin server). There is also the possibility to use the Jade as stateless signing device only. 
+
+<p align="center">
+<img src="/assets/img/jade.png" class=responsive width="350" height="350" maxheight="350">
```
Comment (BitcoinQnA) [inline wallet/hardware.md:91]: Can you get an image with the background removed so it matches the style of the page please?

I like to use a tool like https://www.remove.bg/

File: wallet/hardware.md [hunk 3]
```diff
@@ -82,6 +83,31 @@ This type of set-up does come with additional hurdles that may be more difficult
...
+</p>
+
+### Notable Features
+* Fully air-gapped operation possible
+* Stateless operation possible
+* Fully open source
+
+
+### Notable Limitations
+* No physical secure element but the possibility to save your seed on the device in combination with a virtual secure element *(not mandatory)*
```
Comment (BitcoinQnA) [inline wallet/hardware.md:101]: Would be good to link to docs on the PIN server here so people fully understand the trade-offs

Comment (marc3linho) [reply wallet/hardware.md]: done with https://github.com/BitcoinQnA/Bitcoiner.Guide/pull/39/commits/200e6a036ca1638bc37953c58d56e7ffea62c896

File: wallet/hardware.md (hunk 2 above)
Comment (marc3linho) [reply wallet/hardware.md]: done with [e9ac1b3](https://github.com/BitcoinQnA/Bitcoiner.Guide/pull/39/commits/e9ac1b39209d9705dd6aa0935799409c910798ea)

File: wallet/hardware.md (hunk 1 above)
Comment (marc3linho) [reply wallet/hardware.md]: done with [e9ac1b3](https://github.com/BitcoinQnA/Bitcoiner.Guide/pull/39/commits/e9ac1b39209d9705dd6aa0935799409c910798ea)

**Comment by BitcoinQnA** [comment]
Thanks again!

**Pull request merged**